### PR TITLE
Delivery: the worktree is a fact, so stop gating it on the workflow catalog

### LINKED ISSUE #126: Delivery: the worktree path is a DTO fact and shouldn't be gated on /workflows
Follow-up from #125, disclosed in that PR rather than fixed in-branch because it needs a design call.

## What happens now

#125 gates the **"no deliver phase"** state on positive knowledge: the Delivery section renders for a run with no deliver unit only when the daemon confirms its workflow is an ordinary (non-`is_system`) catalog def. That was the right fix for the *claim* — 86 of 129 live runs carry a materialised `wf-<runId>` id that `GET /workflows` never serves, so "this run has no deliver phase" was being asserted about runs studio cannot classify.

But the gate took a **fact** down with the claim. For a catalog `feature` run with no deliver phase, the section also carries the **worktree path**, which comes straight off the run's own DTO (`session.workdir`) and is true regardless of what workflow produced it. Today that path is withheld whenever the classification is unavailable:

| `/workflows` state | what the operator sees |
|---|---|
| loaded | section + worktree + remedy |
| still loading | **nothing**, then it pops in a tick later |
| request failed | **nothing at all** |

**24 live runs** are in this class. The cold-load pop-in is visible jitter on every navigation, and the failed-fetch case means a daemon hiccup silently removes a surface that has nothing to do with `/workflows`.

## The split

Verification specified the fix precisely, and I agree with it:

- Render the section, with the **worktree line**, whenever `session.workdir` is known — no lookup required, because it is a DTO fact.
- Gate **only** the "this run has no deliver phase" sentence and the `deliver: pr` remedy on `isSystemWorkflow(id) === false`.

That keeps the honesty property #125 exists to establish (never assert a classification the wire cannot evidence) while not withholding something the wire *does* evidence.

## Why it wasn't done in #125

It changes what the `'none'` state means — from one gated block to a fact-block plus a gated sentence — and #125 had already been through four verification rounds. Folding a design change into the fifth would have meant re-running the whole live-corpus measurement to prove the delta again. Cleaner as its own slice with its own before/after.

## Acceptance

- A catalog `feature` run with no deliver unit renders `run-delivery` with the worktree line in **all three** cache states (warm, still-loading, failed) — asserted in the DOM, not reasoned about.
- The "no deliver phase" sentence and the `deliver: pr` remedy appear **only** when `isSystemWorkflow(id) === false`.
- An `is_system` run (e.g. `interactive-draft`) still renders no section at all, warm or cold.
- No new requests on any list surface; the right panel's budget is unchanged.
- No cold-load pop-in for a run whose `workdir` is known.

Related: #122 (the slice), and wicked-crew#321 — once `session.delivery` is on the list wire, more of this resolves without a lookup at all.

### PR DESCRIPTION
**Pull request by mikeparcewski**
Closes #126.

#125 gated the "no deliver phase" **claim** on positive knowledge — right — but withheld the whole section to do it, taking the **worktree path** down with the sentence. That path is `session.workdir`, straight off the run's own DTO: true whatever composed the run, knowable with no lookup.

Measured on the live daemon: **24 runs** are in this class. The section popped in a tick after every navigation, and vanished outright when `GET /workflows` failed — a daemon hiccup removing a surface that doesn't depend on the daemon's workflow list.

## The split

| | gated on |
|---|---|
| the section + **worktree line** | `session.workdir` is known — a DTO fact (`hasDeliverySection`, new) |
| "This run has no deliver phase." | `is_system === false` (unchanged, inside `RunDelivery`) |
| `deliver: pr` remedy | `is_system === false` (unchanged) |

`canDeliver` is deliberately **not** widened. The census and row chips keep the narrower predicate — widening them would put unclassifiable runs back into a "N no deliver phase" census line, the exact D5 claim #125 removed.

## The denylist gap this exposed

Rendering on a fact instead of the catalog makes *"show a `feature` run's worktree while the defs load"* and *"never show an `interactive-draft` run a section, warm or cold"* the **same cold-cache instant**. The denylist named 5 of the daemon's 11 system workflows, so studio had nothing to tell them apart by — #126's acceptance criteria were unsatisfiable until it closed.

Measured (17 defs, 11 `is_system: true`) and closed: the denylist now names all eleven. Adding an id can only ever **withhold** delivery (`deliverKindOf` is one-directional), and each is one of crew's own reserved names.

## A fixture that misdescribed the runs it named

`docThread()` built the three real `DOC_THREAD_RUN_IDS` with `workdir: '/w/tree'`. All three report **`workdir: null`** live — a document thread writes into the interactive doc workspace, not a run worktree. That field had never mattered; under a fact-gate it would have modelled those runs as gaining a section the real ones cannot.

It's also what keeps EC57 self-limiting: **no worktree → no fact → no section**. Document threads stay silent for a structural reason, not a lucky one.

## Acceptance — asserted in the DOM

- [x] worktree line renders **warm**, **still-loading**, and **failed-fetch**
- [x] claim + remedy appear only when `isSystemWorkflow(id) === false`
- [x] an `is_system` run renders **no section at all**, any cache state
- [x] a run with **no workdir and no licence** renders nothing (a *licensed* run with no workdir still states the phase fact and the remedy — deliberate since #125, pinned by `delivery.coldCache.test.tsx:197`)
- [x] no new requests on any list surface

1823/1823 studio tests green (180 files), tsc + lint + build clean.

### CODE REVIEW

File: src/components/delivery.ts [hunk 1]
```diff
@@ -370,6 +370,58 @@ export function canDeliver(view: SessionView, isSystemWorkflow?: IsSystemWorkflo
...
+export function hasDeliverySection(
+  view: SessionView,
+  isSystemWorkflow?: IsSystemWorkflow,
+): boolean {
+  if (canDeliver(view, isSystemWorkflow)) return true;
+  const wf = view.session.workflow_id?.trim() ?? '';
+  if (deliverKindOf(wf, isSystemWorkflow) !== 'build') return false;
+  const workdir = view.session.workdir;
+  return typeof workdir === 'string' && workdir.trim() !== '';
+}
```
[the remark this answers is not in the document]
Comment (mikeparcewski) [reply src/components/delivery.ts:436]: Correct that the two disagree — the wording is what's wrong, not the gate.

A run whose workflow is *proven ordinary* (`is_system === false`) and has no `workdir` renders the section deliberately, and has since #125: the "This run has no deliver phase." sentence and the `deliver: pr` remedy are both **licensed** there, so both are evidenced. `tests/delivery.coldCache.test.tsx:197` ("a run with no workdir says the phase fact and the remedy, and nothing else") pins exactly that, and this PR doesn't touch it. Gating that arm on `workdir` would withhold a claim the wire *does* evidence — the mirror of the bug #126 exists to fix.

The three arms are each independently evidenced:

| arm | evidence |
|---|---|
| has a deliver phase | the unit itself |
| `is_system === false` | a def in hand (the claim is licensed) |
| build-kind + known `workdir` | the DTO fact (**new** — this is the flicker fix) |

So no path renders on nothing. What was wrong is my acceptance bullet: I wrote "a run with no workdir renders nothing", which is only true for a run that *also* lacks the licence — the doc-thread case my new test asserts (`r-doc-thread`, materialised id + `workdir: null` → no section, warm or cold).

Fixed the PR body to say "no workdir **and** no licence". Thanks — the sentence would have misled the next reader into "correcting" the code and regressing #125.
